File: WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/SitePickerPage.java
package org.wordpress.android.e2e.pages;

import static androidx.test.espresso.Espresso.onView;
import static androidx.test.espresso.matcher.ViewMatchers.withText;
import static org.wordpress.android.support.WPSupportUtils.clickOn;

public class SitePickerPage {
    public SitePickerPage() {
    }

    public void chooseSiteWithURL(String url) {
        clickOn(onView(withText(url)));
    }
}

File: WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/SitePickerPage.kt
package org.wordpress.android.e2e.pages

import androidx.test.espresso.Espresso
import androidx.test.espresso.matcher.ViewMatchers
import org.wordpress.android.support.WPSupportUtils

class SitePickerPage {
    fun chooseSiteWithURL(url: String?) {
        WPSupportUtils.clickOn(Espresso.onView(ViewMatchers.withText(url)))
    }
}
